### DISCUSSION TITLE
./ptx_ir.h:1053: int ptx_instruction::get_type2() const: Assertion `m_scalar_type.size() == 2' failed.

### DISCUSSION THREAD

**Discussion by eecspan** [accel-sim/accel-sim-framework#303] [Q&A]
I use the gpgpusim in accel-sim, when i want to run cutlass-bench/examples/05_wmma_gemm with turing architecture, I encountered the issue "Aborted (core dumped). The detailed error messages are: “./ptx_ir.h:1053: int ptx_instruction::get_type2() const: Assertion `m_scalar_type.size() == 2' failed.” and "error: set_wmma_vector_operands".
Before this, I had already resolved some issues, which were mentioned in https://github.com/gpgpu-sim/gpgpu-sim_distribution/issues/296
Is this issue occurring because GPGPU-Sim does not yet support using tensor cores for s8 matrix multiplication and can only perform float16 matrix multiplication?
Thank you very much. I hope you can provide assistance.

**Comment by JRPan** [comment]
I think lots of Turing+ features are not supported in gppgu-sim PTX mode. To support that, change the front end to parse for the s8 identifier in the PTX instruction.

**Reply by eecspan** [reply]
OK, thank you a lot!  I understand！
